### PR TITLE
AsterioBid Analytics Adapter : initial release

### PR DESCRIPTION
**Pull request by danoykin**
## Type of change
New bidder adapter

## Description of change
admin@asteriobid.com

pbjs.enableAnalytics({
    provider: 'asteriobid',
    options: {
        bundleId: '04bcf17b-9733-4675-9f67-d475f881ab78'
    }
});

### CODE REVIEW

File: modules/asteriobidAnalyticsAdapter.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Overview
+
+Module Name: AsterioBid Analytics Adapter
+Module Type: Analytics Adapter
+Maintainer: admin@asteriobid.com
+
+# Description
+Analytics adapter for <a href="https://asteriobid.com/">AsterioBid</a>. Contact admin@asteriobid.com for information.
```
Comment (ncolletti) [inline modules/asteriobidAnalyticsAdapter.md:8]: Could you include information about your config options?

**Review by ncolletti — COMMENTED** [review]
What are the major differences between your existing analytics adapter, prebidmanager? Do you intend to point your customers to one particular adapter moving forward?

File: modules/asteriobidAnalyticsAdapter.md (hunk 1 above)
Comment (danoykin) [reply modules/asteriobidAnalyticsAdapter.md:8]: I added the description of test and advanced configuration parameters

**Comment by danoykin** [comment]
> What are the major differences between your existing analytics adapter, prebidmanager? Do you intend to point your customers to one particular adapter moving forward?

There are performance improvements and the support of new features in 'asteriobid' adapter. We are going to switch all customers from 'prebidmanager' to 'asteriobid' adapter. The main cause is rebranding of PrebidManager product to AsterioBid.

**Review by ncolletti — APPROVED** [review]
lgtm

**Pull request merged**